drivers/net/ethernet/microchip/enc28j60.c

```diff
@@ -1294,7 +1294,13 @@ static netdev_tx_t enc28j60_send_packet(struct sk_buff *skb,
 	netif_stop_queue(dev);
 
 	/* Remember the skb for deferred processing */
-	priv->tx_skb = skb;
```

Comment:
@iluminat23  @linosanfilippo-kunbus there is no queue for the xmit in this drive. Any idea about why it was not needed, and whether it is needed for our piBridge?

Reply:
yes, here I mean the driver specific list of skbuff.